ckalkan/kalkan.go

```diff
@@ -1,11 +1,23 @@
 package ckalkan
 
 import (
+	"runtime"
 	"sync"
 )
 
+func getLibraryName() string {
+	switch runtime.GOOS {
+	case "freebsd":
+		return "libkalkancryptwr-64.so.2"
+	case "linux":
+		return "libkalkancryptwr-64.so"
+	default:
+		panic("GOOS=" + runtime.GOOS + " is not supported")
+	}
+}
```

Comment:
What's the point of `libkalkancryptwr-64` library differently on freebsd machines? I mean, why not just to use the same name as `libkalkancryptwr-64.so`?

Reply:
libkalkancryptwr-64 on linux uses glibc, on freebsd own implementation of C Library or must use linux emulation, and this creates its own problems.
On freebsd machines  lib name must end of number(e.g *.so.2), otherwise ldconfig won't see it. 
That's why it was necessary to compile the library separately on freebsd.

Reply:
Ok, great.